seeds.md

```diff
@@ -0,0 +1,88 @@
+# Seeding the Application
+
+For development and testing purposes a developer may want to work with a large data set. If they were forced to achieve a large dataset by creating each record one-by-one it would waste valuable time and effort. The solution is [seeding the database](https://en.wikipedia.org/wiki/Database_seeding). To make this process even more effecient (_and fun_) we will be utilizing the [Faker](https://github.com/faker-ruby/faker) gem.
+
+## Step 1 - Installing Faker
+
+To start using `Faker`, first add it to your `Gemfile`: 
```

Comment:
Thinking about this, anytime we mention a gem, we should probably link to it. Drive folks to the authority.

```suggestion
To start using [`Faker`](https://github.com/faker-ruby/faker), first add it to your `Gemfile`: 
```